pom.xml

```diff
@@ -62,6 +82,11 @@
             <artifactId>junit-platform-surefire-provider</artifactId>
             <version>1.2.0</version>
           </dependency>
+          <dependency>
+			  <groupId>org.junit.jupiter</groupId>
+              <artifactId>junit-jupiter-engine</artifactId>
+              <version>5.2.0</version>
+          </dependency>
```

Comment:
did we need to include all of these imports? we already had a working junit setup.

Reply:
I need them for migration support.

Reply:
Migration support?

Reply:
Yes. This is your comment in the code;

[* FIXME: will wait for junit-pioneer to add TempDirectory for this test](https://github.com/in-toto/in-toto-java/blob/master/src/test/java/io/in_toto/keys/RSAKeyTest.java)

But I used TemporaryFolder from junit4, and I need these dependencies to use it in junit5 in the [tests](https://github.com/m0jt4b4/in-toto-java/blob/master/src/test/java/io/in_toto/models/LinkTest.java).

Reply:
Yes, that's part of the problem. junit-pioneer will add `TempDirectory` to Junit5. I guess this is fine for now, but let's leave that comment, as I do want to use Junit5...

Reply:
Yes, it's better to use only juinit5 in future.